security/vertx-jwt/src/main/java/io/quarkus/ts/security/vertx/services/AbstractRedisDao.java

```diff
@@ -10,17 +10,17 @@
 
 import jakarta.inject.Inject;
 
-import io.quarkus.redis.client.reactive.ReactiveRedisClient;
 import io.quarkus.ts.security.vertx.exceptions.NotFoundException;
 import io.quarkus.ts.security.vertx.model.Record;
 import io.smallrye.mutiny.Multi;
 import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.redis.client.RedisAPI;
 import io.vertx.mutiny.redis.client.Response;
 
 public abstract class AbstractRedisDao<E extends Record> {
 
     @Inject
-    ReactiveRedisClient redisClient;
+    RedisAPI redisClient;
```

Comment:
Is this equivalent change from reactive perspective. Can you please confirm it?

Reply:
`RedisAPI` is implementing both [reactive and imperative API](https://quarkus.io/guides/redis-reference#apis). But imperative methods are suffixed with string like "andWait" (e.g. method `ftAliasaddAndAwait(...)` vs. reactive one `ftAliasadd(...)`).
So yes `RedisAPI` should be equivalent from reactive perspective.

Reply:
Ok, thanks for explanation